build-system/compile/compile.js

```diff
@@ -394,7 +478,6 @@ function compile(
       timeInfo.startTime = Date.now();
       return gulp
         .src(gulpSrcs, {base: SRC_TEMP_DIR})
-        .pipe(gulpIf(shouldShortenLicense, shortenLicense()))
```

Comment:
Does this mean the polymer and bsd licenses are no longer included in the code?

Assuming that's true, I think the removal of shortenLicense should be moved to a separate PR in order to make future tracking / maintenance of this file easier.

Reply:
The licenses are still included in the code, the work done during the `shortenLicense` step was removed and done to the local copy of the code (with a comment in the alterations sections to address this change).

Possible to remove from this PR, but it's quite relevant to the babel work related to comments included below.

Reply:
SGTM!